### DISCUSSION TITLE
How do I stream videos from a server folder?

### DISCUSSION THREAD

**Discussion by romanrsr** [owncast/owncast#3521] [Q&A] [answered]
Assuming I have a folder with hundreds of videos on the same server where owncast is installed, how do I configure it to play those videos?

**Comment by aaronpk** [comment, marked as answer]
Owncast is a viewer for a livestream, so you can use whatever tools you want to create the stream in the first place. There are lots of open source and proprietary software for scheduling video playback depending on what kind of features you want. Also look up TV automation software. A low tech option would also be ffmpeg like this https://video.stackexchange.com/questions/29375/loop-all-videos-from-folder-to-rtmp-stream

**Comment by gabek** [comment]
Just like you would with a live stream, you'd still need to "broadcast" the video from somewhere to Owncast. There's a small demo project at https://github.com/owncast/playout-example that shows how you could put something like this together. It's just an example, but hopefully, it might point you in the right direction of how you could use playout or playback automation software with Owncast.
